scalafix-tests/src/test/resources/checkSyntax/NoAutoTupling.source

```diff
@@ -0,0 +1,89 @@
+rewrites = [NoAutoTupling]
```

Comment:
Ah right, I've seen that issue. Ok, I'll leave it like this, so that it can be fixed together with the rest.